ci/run_fuzzit.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -eux
+# Copyright 2019 fuzzit.dev Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+
+function abspath {
+    if [[ -d "$1" ]]
+    then
+        pushd "$1" >/dev/null
+        pwd
+        popd >/dev/null
+    elif [[ -e $1 ]]
+    then
+        pushd "$(dirname "$1")" >/dev/null
+        echo "$(pwd)/$(basename "$1")"
+        popd >/dev/null
+    else
+        echo "$1" does not exist! >&2
+        return 127
+    fi
+}
+
+# Uploading to Fuzzit
+export FUZZIT_ARGS="--type $1"
+wget -O fuzzit https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.30/fuzzit_Linux_x86_64
+chmod a+x fuzzit
+
+PREFIX="`abspath /build/tmp/_bazel_bazel/*/execroot/envoy/bazel-out/k8-fastbuild/bin/test`"
+
+./fuzzit create job ${FUZZIT_ARGS} envoyproxy/access-log-formatter                      $PREFIX/common/access_log/access_log_formatter_fuzz_test_with_libfuzzer
```

Comment:
Can you do something with Bazel query or `find` to make it automatically pick up the fuzzers that should be run? This makes it more maintainable as we add/remove fuzzers. See https://github.com/google/oss-fuzz/blob/master/projects/envoy/build.sh#L56 and https://github.com/envoyproxy/envoy/blob/master/ci/do_ci.sh#L258 for how this can be done.

Reply:
I'm not sure as we need to define a target for each fuzzer on fuzzit beforehand. I mean I guess it is possible to do some kind of more automation here but I'm a bit unsure here. How often new fuzz targets are added?

Reply:
Maybe one every few weeks, we are trying to encourage a lot more development here in general. Also, some the build targets have information such as `nofuzz` which tells us which fuzzers we want to skip (e.g. they are too trivial to be worth running). I recommend making this kind of discovery more dynamic, as you reduce friction to adding/removing/disabling fuzzers.

Reply:
Gotcha. ok I think I can make something along those lines.

Reply:
@htuch done! Now the targets are auto sourced with bazel as well as auto created on Fuzzit!